packages/p2p/src/stores/buy-sell-store.js

```diff
@@ -321,90 +279,13 @@ export default class BuySellStore extends BaseStore {
-                p2p_advert_list: 1,
-                counterparty_type,
-                offset: startIndex,
-                limit: general_store.list_item_limit,
-                sort_by: this.sort_by,
-                use_client_limits: this.should_use_client_limits ? 1 : 0,
-                ...(this.selected_payment_method_value.length > 0
-                    ? { payment_method: this.selected_payment_method_value }
-                    : {}),
-                ...(this.selected_local_currency ? { local_currency: this.selected_local_currency } : {}),
-            }).then(response => {
-                if (response) {
-                    if (!response.error) {
-                        // Ignore any responses that don't match our request. This can happen
-                        // due to quickly switching between Buy/Sell tabs.
-                        if (response.echo_req.counterparty_type === counterparty_type) {
-                            const { list } = response.p2p_advert_list;
-
-                            this.setHasMoreItemsToLoad(list.length >= general_store.list_item_limit);
-
-                            const old_items = [...this.items];
-                            const new_items = [];
-
-                            list?.forEach(new_item => {
-                                const old_item_idx = old_items.findIndex(old_item => old_item.id === new_item.id);
-
-                                if (old_item_idx > -1) {
-                                    old_items[old_item_idx] = new_item;
-                                } else {
-                                    new_items.push(new_item);
-                                }
-                            });
-
-                            this.setItems([...old_items, ...new_items]);
-
-                            const search_results = [];
-
-                            if (this.search_term) {
-                                this.items.forEach(item => {
-                                    if (
-                                        item.advertiser_details.name
-                                            .toLowerCase()
-                                            .includes(this.search_term.toLowerCase().trim())
-                                    ) {
-                                        search_results.push(item);
-                                    }
-                                });
-                            }
-
-                            if (search_results.length) {
-                                this.setSearchResults(search_results);
-                            } else {
-                                this.setSearchResults([]);
-                            }
-                        }
-                        // Added a check to prevent console errors
-                    } else if (response && response.error.code === api_error_codes.PERMISSION_DENIED) {
-                        this.root_store.general_store.setIsBlocked(true);
```

Comment:
just a doubt.. where will this condition be handled?

Reply:
its in useP2PRenderedAdverts 👀 a lot of the logic was moved into the hooks itself

Reply:
Nice catch @nada-deriv, I forgot to get back to this. Will handle it

Reply:
@nada-deriv, I had forgotten to follow up on this, but I tested this and it had been handled already in the `general_store.updateAdvertiserInfo`. This was a precautionary check I guess and was redundant. As you can see in the screenshot, when that error is thrown, the blocked page is displayed.

![error](https://github.com/binary-com/deriv-app/assets/104334373/6f3bea74-1f4b-4c9d-b54f-38d6632e7484)